cmd/syncthing/gui.go

```diff
@@ -302,6 +302,15 @@ func restGetModel(m *model.Model, w http.ResponseWriter, r *http.Request) {
 	res["state"], res["stateChanged"] = m.State(folder)
 	res["version"] = m.CurrentLocalVersion(folder) + m.RemoteLocalVersion(folder)
 
+	ignorePatterns, _, _ := m.GetIgnores(folder)
+	nonTrivialPattern := false
+	for _, line := range ignorePatterns {
+		if len(line) > 0 && !strings.HasPrefix(line, "//") {
+			nonTrivialPattern = true
```

Comment:
I guess you could break out of iteration once this becomes true, as there is no point to continue the iteration.
Also, you could assign the value directly to `res["ignorePatterns"]`